### ISSUE TITLE
recent autotools requires 'subdir-objects'

### ISSUE THREAD

**Issue by dkj** [wtsi-npg/bambi#49]
Running ```autoreconf -if```, to generate the configure file, on Xenial and Yakkety errors out.

Seems to need:
```
AM_INIT_AUTOMAKE([subdir-objects -Wall -Werror foreign tar-pax no-dependencies])
```
rather than
```
AM_INIT_AUTOMAKE([-Wall -Werror foreign tar-pax no-dependencies])
```
in configure.ac

**Comment by jenniferliddle** [comment]
Making this change causes Travis to fail with an error:
```
gcc -DHAVE_CONFIG_H -I. -I../../test -I..  -I/home/travis/build/wtsi-npg/bambi/htslib/b/include  -std=gnu99 -Werror  -I../../src -DDATA_DIR=/home/travis/build/wtsi-npg/bambi/bambi-0.9.5-8-g50a40f0/_build/../test/data -g -O2 -c -o t_read2tags-t_read2tags.o `test -f 't_read2tags.c' || echo '../../test/'`t_read2tags.c

/bin/bash: ../../src/.dirstamp: Permission denied
```
More investigation required.

**Comment by jenniferliddle** [comment]
Fixed with PR #79
